plugin/src/main/kotlin/com/github/triplet/gradle/play/ProcessPackageMetadataTask.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.triplet.gradle.play
+
+import com.android.build.gradle.api.ApkVariantOutput
+import com.github.triplet.gradle.play.internal.PlayPublishTaskBase
+import com.github.triplet.gradle.play.internal.ResolutionStrategy
+import org.gradle.api.tasks.TaskAction
+
+open class ProcessPackageMetadataTask : PlayPublishTaskBase() {
+    @TaskAction
+    fun processVersionCodes() = read { editId ->
+        val maxVersionCode = bundles().list(variant.applicationId, editId).execute().bundles
+                ?.map { it.versionCode }
+                ?.max() ?: apks().list(variant.applicationId, editId).execute().apks
+                ?.map { it.versionCode }
+                ?.max() ?: 1
+
+        when (extension._resolutionStrategy) {
+            ResolutionStrategy.AUTO -> variant.outputs.filterIsInstance<ApkVariantOutput>().forEach {
+                val newCode = maxVersionCode + 1
+                it.versionCodeOverride = newCode
+                it.versionNameOverride = "${variant.versionName}.$newCode"
```

Comment:
Is this something that should be configurable? I know not everybody uses this naming convention.